lib/components/artist/related_artists/related_artist.js

```diff
@@ -49,7 +52,7 @@ const styles = StyleSheet.create({
 });
 
 export default Relay.createContainer(RelatedArtist, {
-  initialVariables: { width: 150, height: 100 },
+  initialVariables: { imageWidth: ~~(imageWidth), imageHeight: ~~(imageWidth / 1.5) },
```

Comment:
I love such tricks, but it’s a bit much and not very readable and readability is more important imo